### ISSUE TITLE
icon no longer changes color with firefox update

### ISSUE THREAD

**Issue by magicanthony** [BoringCode/RedditNotifier#31]
I have made several firefox updates lately and a few updates ago the icon stopped changing color when I get new messages.  It still makes a sound but no longer changes color.  This has been happening for quite a while now and I'd like to know when this will be fixed.  My current firefox version is 38.0.1 but it started happening several versions ago.

Thank you.

**Comment by BoringCode** [comment]
Yeah, I'm aware of this problem and will be looking into a fix in the coming weeks.

**Comment by magicanthony** [comment]
My reddit notifier just updated to the new version (1.5.4.1) and the  issue is still not fixed even though it said it was fixed in the release notes.  I tried clearing all history, cookies, etc. in firefox, restarting firefox and relogging into reddit.  It still plays the sound but it does not change color when I get new messages.  I have the latest version of firefox (38.0.1).  When will this annoying issues be fixed ??
